src/admin/components/SearchForm.js

```diff
@@ -1,31 +1,31 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 
+import { SearchFormBase } from 'amo/components/Search/SearchForm';
 import { fetchAddon } from 'core/api';
 import { loadEntities } from 'core/actions';
 import { gettext as _ } from 'core/utils';
 
-import 'admin/css/SearchForm.scss';
-import 'admin/css/lib/buttons.scss';
+import 'amo/css/SearchForm.scss';
+import 'amo/css/lib/buttons.scss';
 
-export class SearchFormBase extends React.Component {
+export class AdminSearchFormBase extends SearchFormBase {
   static propTypes = {
     api: PropTypes.object.isRequired,
     loadAddon: PropTypes.func.isRequired,
     pathname: PropTypes.string.isRequired,
-    query: PropTypes.string.isRequired,
+    query: PropTypes.string,
```

Comment:
Yeah... although it still runs fine with the `isRequired` there... I don't remember making this change so I should probably back it out.